src/db/connection.js

```diff
@@ -1,5 +1,5 @@
 const { Sequelize } = require("sequelize")
-const connection = new Sequelize(process.env.DB_URI);
+const connection = new Sequelize(process.env.MYSQL_URI);
```

Comment:
Change this in your env file instead of in the code - the name here is not important but this change will break it for everyone else